gulpfile.js

```diff
@@ -449,6 +449,9 @@ gulp.task('buildnumber', function (done) {
     if (!err) {
       // Build number is the number of commits since base version
       buildNumber = stdout ? stdout.match(/\n/g).length : 0;
+    } else {
+      console.log('This is not a Git repository.');
+      console.log('Using default build number.');
```

Comment:
This should fit on *one* line, i.e. format it as `console.log('This is not a Git repository; using default build number.');` which is what https://github.com/mozilla/pdf.js/pull/9852#discussion_r198991482 asked for.

You also need to squash the commits, see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.